tests/features/steps/artifacts-poetry-plugin-steps.py

```diff
@@ -15,52 +13,107 @@
 
 
 class Package:
-    def __init__(self, name, pretty_version) -> None:
+    def __init__(self, name, pretty_version, files) -> None:
         self.name = name
         self.pretty_version = pretty_version
+        self.files = files
 
     def name(self):
         return self.name
 
     def pretty_version(self):
         return self.pretty_version
 
+    def files(self):
+        return self.files
+
 
 @given("the following artifacts")
 def the_following_artifacts(context):
     context.packages = {}
     for row in context.table:
+        name = row["package"]
+        version = row["version"]
+        filename = f"{name}-{version}.whl"
+        file = open(os.path.join(TEST_RESOURCES_DIR, filename), "w")
+        file.close()
+        files = []
+        files.append(filename)
```

Comment:
A:
```suggestion
        files = [filename]
```

Reply:
Yeah it's super minor and it's totally fine if we don't change it. Just seemed a little (dare I say it? is Ryan looking?) "non-pythonic" to create an empty list and then append vice just create the list you want. At the end of the day it's really a matter of taste anyway.